pkg/interface/src/logic/reducers/hark-update.ts

```diff
@@ -187,7 +187,14 @@ function readSince(json: any, state: HarkState): HarkState {
 
 function unreadSince(json: any, state: HarkState): HarkState {
   const data = _.get(json, 'unread-count');
-  if(data) {
+  if (data) {
+    if (!useHarkState.getState().doNotDisturb && !document.hasFocus()) {
+      new Notification('New Landscape Notification', {
+        tag: 'landscape',
```

Comment:
? This will notify on literally every message lol. Move this logic to 'add-note'